public/js/add.js

```diff
@@ -0,0 +1,40 @@
+function addEmployeeField() {
```

Comment:
Whole file: Change to camelcase for variable names

Reply:
add.js is Isis Task, just pulled for client-side testing

Reply:
alles klar